src/components/BasicSide.vue

```diff
@@ -133,27 +133,49 @@ export default {
       menuViewList.push(menuView)
     }
     this.menuViewList = menuViewList
-    this.menuView = menuViewList[0]
+    console.log("menuViewList", this.menuViewList)
+
+    if(this.$route.name != this.custom.special.view) {
```

Comment:
this logic only handles two views - it needs to handle 1..n views

Reply:
Done!